rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction-reacttodifferentoutcomes.adoc

```diff
@@ -0,0 +1,15 @@
+# React to Different Outcomes
+
+In the model of the process you used an exclusive gateway to determine how to proceed with different outcomes of the tracking number extraction.
```

Comment:
The first paragraph should be the short description, which summarizes the topic.
https://confluence.internal.salesforce.com/x/5LmeBw